lib/components/search_user/interactor/mappers/search_user_state_mapper.dart

```diff
@@ -4,16 +4,31 @@ import 'package:seeds/domain-shared/page_state.dart';
 import 'package:seeds/domain-shared/result_to_state_mapper.dart';
 
 class SearchUserStateMapper extends StateMapper {
-  SearchUserState mapResultToState(SearchUserState currentState, Result result, List<String>? noShowUsers) {
-    if (result.isError) {
+  SearchUserState mapResultToState(
+      SearchUserState currentState, Result seedsMembersResult, Result telosResult, List<String>? noShowUsers) {
+    if (seedsMembersResult.isError && telosResult.isError) {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: 'Error Searching for User');
     } else {
-      final List<MemberModel> users = result.asValue?.value as List<MemberModel>;
-      if (noShowUsers != null) {
-        for (final noShowUser in noShowUsers) {
-          users.removeWhere((element) => element.account == noShowUser);
+      List<MemberModel> seedsUsers = [];
+      List<MemberModel> telosUsers = [];
+
```

Comment:
the logic for seeds users is the same as before. 

Not quite sure what no show users are?!

Reply:
Which means we should probably remove the active user from the telos accounts as well, "If its coming back" in the response

Reply:
Got it

Reply:
I made that change - no show now applies to the final list